nyaa/templates/layout.html

```diff
@@ -6,9 +6,9 @@
 
 		<meta name="viewport" content="width=480px">
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
-		<link rel="shortcut icon" type="image/png" href="{{ url_for('static', filename='favicon.png') }}">
-		<link rel="icon" type="image/png" href="{{ url_for('static', filename='favicon.png') }}">
-		<link rel="mask-icon" href="{{ url_for('static', filename='pinned-tab.svg') }}" color="#3582F7">
+		<link rel="shortcut icon" type="image/png" href="/static/favicon.png">
```

Comment:
this has 0 actual performance impact but requires us to manually hardcode a path once we get a CDN domain for static resources, instead of just being able to register that one view to be the CDN domain.

Pointless and dumb